crates/edr_provider/src/data.rs

```diff
@@ -2521,11 +2561,14 @@ mod tests {
         let pending_transaction = fixture
             .provider_data
             .sign_transaction_request(transaction)?;
-        fixture
-            .provider_data
-            .run_call(pending_transaction, None, &StateOverrides::default())?;
 
-        let console_log_calls = fixture.console_log_calls();
+        let result = fixture.provider_data.run_call(
+            pending_transaction,
+            None,
+            &StateOverrides::default(),
+        )?;
+
+        let console_log_calls = result.console_log_inputs;
```

Comment:
Nit: if you make changes anyway, could you rename console_log_calls to console_log_inputs? Otherwise feel free to ignore

Reply:
Addressed in https://github.com/NomicFoundation/hardhat/pull/4781/commits/498f131e4d40a3a8b697bdcfc68c6c8ea6e700db